lib/js_defer.js

```diff
@@ -1613,58 +1450,42 @@ deferJsNs.DeferJs.prototype.noDeferCreateElementOverride = function() {
         }
       };
       deferJsNs.addOnload(elem, onload);
-      pagespeedutils.addHandler(elem, 'error', onload);
+      fasterizeutils.addHandler(elem, 'error', onload);
     }
     return elem;
   };
 };
 
 /**
- * @return {boolean} true if deferJs is running with experimental flag.
- */
-deferJsNs.DeferJs.prototype.isExperimentalMode = function() {
-  return deferJsNs.DeferJs.isExperimentalMode;
-};
-deferJsNs.DeferJs.prototype['isExperimentalMode'] =
-    deferJsNs.DeferJs.prototype.isExperimentalMode;
-
-/**
  * Initialize defer javascript.
  */
 deferJsNs.deferInit = function() {
-  if (pagespeed['deferJs']) {
+  if (fasterize['deferJs']) {
     return;
   }
 
-  deferJsNs.DeferJs.isExperimentalMode = pagespeed['defer_js_experimental'];
-  pagespeed.highPriorityDeferJs = new deferJsNs.DeferJs();
-  pagespeed.highPriorityDeferJs.setType(
-    deferJsNs.DeferJs.PRIORITY_PSA_SCRIPT_TYPE);
-  pagespeed.highPriorityDeferJs.setPsaNotProcessed(
-    deferJsNs.DeferJs.PRIORITY_PSA_NOT_PROCESSED);
-  pagespeed.highPriorityDeferJs.setNotProcessedAttributeForNodes();
-  pagespeed.lowPriorityDeferJs = new deferJsNs.DeferJs();
-  pagespeed.lowPriorityDeferJs.setType(deferJsNs.DeferJs.PSA_SCRIPT_TYPE);
-  pagespeed.lowPriorityDeferJs.setPsaNotProcessed(
+  fasterize.deferJs = new deferJsNs.DeferJs();
+  fasterize.deferJs.setType(deferJsNs.DeferJs.PSA_SCRIPT_TYPE);
+  fasterize.deferJs.setPsaNotProcessed(
     deferJsNs.DeferJs.PSA_NOT_PROCESSED);
-  pagespeed.lowPriorityDeferJs.setNotProcessedAttributeForNodes();
-  pagespeed.deferJs = pagespeed.highPriorityDeferJs;
+  fasterize.deferJs.setNotProcessedAttributeForNodes();
 
-  pagespeed.deferJs.noDeferCreateElementOverride();
-  pagespeed['deferJs'] = pagespeed.deferJs;
+  fasterize.deferJs.noDeferCreateElementOverride();
+  fasterize.deferJs.deferInsertScriptTag();
+  fasterize['deferJs'] = fasterize.deferJs;
 };
 
 /**
  * Indicates if deferJs started executing.
  * @type {boolean}
  */
-pagespeed.deferJsStarted = false;
+fasterize.deferJsStarted = false;
 
 /**
  * Starts deferJs execution.
  */
 deferJsNs.startDeferJs = function() {
-  if (pagespeed.deferJsStarted || pagespeed['panelLoader']) return;
+  if (fasterize.deferJsStarted || fasterize['panelLoader']) return;
```

Comment:
I think that we can remove the `panelLoader` check as we do not implement this pagespeed's functionality.